packages/core/src/fhir-deduplication/shared.ts

```diff
@@ -11,6 +11,23 @@ export const SNOMED_OID = "2.16.840.1.113883.6.96";
 export const ICD_10_CODE = "icd-10";
 export const ICD_10_OID = "2.16.840.1.113883.6.90";
```

Comment:
nit: we might want to have all this stuff in a higher level package in core? Theres another place we define these constants. Might be good to have them be fully shared.